file-formats/apob/type/zif_aff_apob_v1.intf.abap

```diff
@@ -0,0 +1,62 @@
+INTERFACE zif_aff_apob_v1
+  PUBLIC.
+  TYPES:
+    "! <p class="shorttext">Application Object Description Type</p>
+    "! Attributes of Application Object Description Type
+    BEGIN OF ty_apob_text,
+      "! <p class="shorttext">Language Code</p>
+      "! Language Code
```

Comment:
Please use sentence case in the descriptions. It is also worth to check the other descriptions here

Reply:
changes as suggested